### PR TITLE
Nv 632 implement backoff for digest

### PR DESCRIPTION
**Pull request by davidsoderberg**

### CODE REVIEW

File: apps/web/src/pages/templates/workflow/DigestMetadata.tsx [hunk 1]
```diff
@@ -76,6 +79,80 @@ export const DigestMetadata = ({ control, index }) => {
...
+        defaultValue="regular"
+        name={`steps.${index}.metadata.type`}
+        render={({ field }) => {
+          return (
+            <Select
+              label="Type of digest"
+              disabled={readonly}
+              placeholder="Regular"
+              data={[
+                { value: 'regular', label: 'Regular' },
```
Comment (davidsoderberg) [inline apps/web/src/pages/templates/workflow/DigestMetadata.tsx:93]: use the enum

File: apps/api/src/app/events/usecases/process-subscriber/process-subscriber.usecase.ts [hunk 2]
```diff
@@ -73,6 +73,21 @@ export class ProcessSubscriber {
...
+          updatedAt: {
+            $gte: from,
+          },
+          _templateId: command.templateId,
+          status: JobStatusEnum.COMPLETED,
+          type: ChannelTypeEnum.TRIGGER,
+          _environmentId: command.environmentId,
+          _subscriberId: subscriber._id,
+        });
+        digestIsRunning = triggerCount > 0;
```
Comment (davidsoderberg) [inline apps/api/src/app/events/usecases/process-subscriber/process-subscriber.usecase.ts:88]: triggerCount < 0;

File: apps/api/src/app/notification-template/dto/update-notification-template.dto.ts [hunk 3]
```diff
@@ -30,6 +30,9 @@ export class NotificationStepDto {
     amount?: number;
     unit?: DigestUnitEnum;
     batchkey?: string;
+    type: DigestTypeEnum;
+    backoffunit?: DigestUnitEnum;
+    backoffamount?: number;
```
Comment (scopsy) [inline apps/api/src/app/notification-template/dto/update-notification-template.dto.ts:35]:
```suggestion
    backoffUnit?: DigestUnitEnum;
    backoffAmount?: number;
```

File: apps/api/src/app/notification-template/dto/update-notification-template.dto.ts [hunk 4]
```diff
@@ -30,6 +30,9 @@ export class NotificationStepDto {
     amount?: number;
     unit?: DigestUnitEnum;
     batchkey?: string;
```
Comment (scopsy) [inline apps/api/src/app/notification-template/dto/update-notification-template.dto.ts:32]:
```suggestion
    batchKey?: string;
```

**Review by scopsy — APPROVED** [review]
A couple of variable naming to be more compliment with our camelCasing, otherwise looks fantastic!

**Pull request merged**